examples/integrated_gradients_transformers.ipynb

```diff
@@ -245,38 +353,43 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "We extract the first transformer's block in the main BERT layer."
+    "We will work with the embedding layer from **distilBERT**. We go with the **embedding layer** since we can easily construct an appropriate baseline for the **IG** that will result in more interpretable attribution. **Note that you can extract any other layer**. Uncomment the preferred one!"
```

Comment:
`We go with the embedding layer since we can easily construct an appropriate baseline for the IG that will result in more interpretable attribution.` 
-> Would be good to add a brief comment on why the attribution would be more interpretable.

Reply:
Yea, it's not clear that the comparison is choosing to work with the embedding layer as opposed to any other intermediate layer. Also we should, separately, say that embedding layer is the standard choice and that we can't attribute to the input because of the discrete structure (phrase it more clearly than I have...).